### ISSUE TITLE
[FEATURE] Make CSS in saved pages prettier than now

### ISSUE THREAD

**Issue by DareFox** [DareFox/SaveDTF-Compose#18]
### Is your feature request related to a problem?

```markdown
Current CSS is ugly and doesn't support embed elements and interactive gallery
```

### Describe the solution you'd like to see

```markdown
Make page look almost like page on DTF/VC/Tjournal
```

### Additional context

@sir-coffee-or-tea created [repo](https://github.com/sir-coffee-or-tea/darefox-dtf-saver-css) with CSS/JS solution to this issue. 

It should be easy to adapt it to the current system

## Current progress: 
|  Element type  | Is supported?  | Since  |
| :------------ | :------------ | :------------ |
|  Title  | 🟩   | **1.0.0**  | 
|  Text |  🟩  |  **1.0.0** |  
|  Image Gallery |  🟩  |  **#19** |  
|  Video Gallery |   🟨 |  **-** |  
|  Image |  🟩  |  **1.0.0** | 
|  Video |  🟩  |  **1.0.0** | 
|  Embed |  🟩  |  **#19** | 
|  Squared text |  🟩  |  **#19** | 
| Quote |  🟨  |  **-** | 
|  Bullet List |  🟩  |  **1.0.0** | 
|  Code |   🟨  |  **-** | 
|  Separator |   🟨  |  **-** | 
|  Audio |  🟩  |  **1.0.0** | 
|  Vote |   🟨  |  **-** | 
|  Number |   🟨   |  **-** | 
|  Person |   🟨  |  **-** | 

- 🟥 - Not supported
- 🟩 - Supported
- 🟨  - Waiting merge

**Comment by DareFox** [comment]
Current development progress of this feature: [feature-css](https://github.com/DareFox/SaveDTF-Compose/tree/feature-css)

**Comment by DareFox** [comment]
Margin 0 breaks single images/videos

https://user-images.githubusercontent.com/47672780/179201206-1b2f42b8-8edc-4451-9128-ce63f66c9702.mp4

**Comment by sir-coffee-or-tea** [comment]
interesting

**Comment by DareFox** [comment]
**Code** element: Official API returns only code in one line without newline breaks
![image](https://user-images.githubusercontent.com/47672780/179841659-6d5dca5b-f9fc-4d4e-b59a-4d8c7dc06f97.png)
![image](https://user-images.githubusercontent.com/47672780/179841853-cc59920c-846d-44b8-a0d3-1bcaca85dfa8.png)

Nothing can be done about it, only parsing from the site itself. For now, I'll do just a stroke in CSS around this element, but I'm unlikely will to full fix it since almost no one uses this element

**Comment by DareFox** [comment]
> Margin 0 breaks single images/videos
> css_margin_issue_crop.mp4

Fixed by c82d8628  ¯\\\_(ツ)\_/¯

**Comment by DareFox** [comment]
Closed by #20
